src/openrct2/Context.cpp

```diff
@@ -848,9 +849,7 @@ namespace OpenRCT2
                     }
 #endif
                 }
-                // This ensures that the newly loaded save reflects the user's
```

Comment:
Why did you move this?

Reply:
It wasn't necessary, but I thought it cleaner to have the logic all _before_ the network logic. Ideally, the network logic will be split off to its own function. As I was touching the line anyway, I thought this was as good a time as any.

Reply:
I’m just afraid it will cause desyncs, so I’d rather it gets moved down again. Apart from this, the PR looks good to me.

Reply:
I'm reasonably confident it won't cause a desync, but I don't feel strongly about the move. I'll move it back down.